openIMIS/openIMIS/settings.py

```diff
@@ -610,8 +623,62 @@ def SITE_URL():
 USE_X_FORWARDED_HOST = True
 SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
 
+OPENSEARCH_DSL = {
+    'default': {
+        'hosts': os.environ.get("OPENSEARCH_HOST", '0.0.0.0:9200'),
+        'http_auth': (
+            f"{os.environ.get('OPENSEARCH_ADMIN')}",
+            f"{os.environ.get('OPENSEARCH_PASSWORD')}"
+        ),
+        'timeout': 120,
+    }
+}
+
+MEDIA_URL = "/file_storage/"
+MEDIA_ROOT = os.path.join(BASE_DIR, "file_storage/")
+
+if not os.path.exists(MEDIA_ROOT):
+    os.makedirs(MEDIA_ROOT)
+
+STORAGES = {
+    "default": {
+        "BACKEND": "django.core.files.storage.FileSystemStorage",
+    },
+    'staticfiles': {
+        'BACKEND': "whitenoise.storage.CompressedManifestStaticFilesStorage",
+    },
+}
+
 PASSWORD_MIN_LENGTH = int(os.getenv('PASSWORD_MIN_LENGTH', 8))
 PASSWORD_UPPERCASE = int(os.getenv('PASSWORD_UPPERCASE', 1))
 PASSWORD_LOWERCASE = int(os.getenv('PASSWORD_LOWERCASE', 1))
 PASSWORD_DIGITS = int(os.getenv('PASSWORD_DIGITS', 1))
 PASSWORD_SYMBOLS = int(os.getenv('PASSWORD_SYMBOLS', 1))
+
+IS_UNIT_TEST_ENV = 'test' in sys.argv
```

Comment:
@sniedzielski  this is used for opensearch SYNC but it should not be used like that there is a opensearch config for that